picard/config.py

```diff
@@ -44,35 +55,42 @@ def __getitem__(self, name):
         return self.value(name, opt, opt.default)
 
     def __setitem__(self, name, value):
+        key = "%s/%s" % (self.__name, name)
         self.lock_for_write()
         try:
-            self.__config.setValue("%s/%s" % (self.__name, name), value)
+            self.__config[key] = value
+            self.__qt_config.setValue(key, value)
         finally:
             self.unlock()
 
-    def __contains__(self, key):
-        key = "%s/%s" % (self.__name, key)
-        return self.__config.contains(key)
+    def __contains__(self, name):
+        key = self.__name + '/' + name
+        return key in self.__config
 
-    def remove(self, key):
-        key = "%s/%s" % (self.__name, key)
-        if self.__config.contains(key):
-            self.__config.remove(key)
+    def remove(self, name):
+        key = self.__name + '/' + name
+        self.lock_for_write()
+        try:
+            if key in self.__config:
+                self.__config.pop(key)
+                self.__qt_config.remove(key)
+        finally:
+            self.unlock()
 
-    def raw_value(self, key):
+    def raw_value(self, name):
```

Comment:
Did this so that there is a consistency between `name` and `key`

Reply:
Ok ;)